docs/_future/getting_started_future.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting started
+order: 1
+slug: getting-started-future
+lang: en
+layout: tutorial
+permalink: /tutorial/getting-started-future
+---
+
+## Getting started <span class="label-beta">BETA</span>
+
+<div class="section-content">
+This guide will cover how to get started with your next-gen platform using Bolt for Python by setting up the Slack CLI and also installing the required dependencies.
+</div>
+
+---
+
+### What is the next-generation platform? {#next-gen}
+
+You can find all the answers to this question on Slack's <a href="https://api.slack.com/future/intro" target="_blank">official introduction page.</a>
+
+<p class="alert alert_info"><ts-icon class="ts_icon_info_circle"></ts-icon>Our next-generation platform is currently in beta. <a href="https://api.slack.com/future/survey" target="_blank">Your feedback is most welcome.</a> - all feedback will help shape the future platform experience!</p>
+
+---
+
+### Setting up the CLI {#setting-up-cli}
+
+To build a next-generation app with Bolt for Python, you'll need to get the Slack CLI.
+
+**Install the Slack Cli** by following this [Quickstart Guide](https://api.slack.com/future/quickstart).
+
+---
+
+### Install dependencies {#install-dependencies}
+
+Once the CLI is set up, make sure your machine has the most recent version of [Python](https://www.python.org/) installed. you can either install Python through a package manager (such as [Homebrew](https://brew.sh/) for Mac) or directly from the website.
+
+---
+
+### Accept the Beta Terms of Service {#accept-tos}
+
+In order to use the next-generation platform features, you'll need to accept a Terms of Service for Slack Platform Beta. To do so, go to the Workspace Settings of the workspace you'll be developing with, then go to the Permissions tab. Find the "Slack Platform Beta" heading, then select the checkbox that says "I have read and agree to the Slack Platform Beta Service Terms" and save the section. Once you have accepted, the section will look like this:
+
+![Beta Terms of Service](https://slack.dev/bolt-js/assets/beta-tos-future.png "Beta Terms of Service")
+
+> 💡 You must be an admin of your workspace to be able to access the Workspace Settings and accept the Terms of Service.
+
+Congratulations! You're now ready to start building using the next-generation Platform. 🎉 You can now proceed with either creating a new app by following the instructions below or <a href="/bolt-python/tutorial/migrate-existing-app" target="_blank">migrating an existing app</a>.
+
+---
+
+### Create a new app {#create-app}
+
+Before you start developing with Bolt, you'll want to create a Slack app.
+
+To create the app, you'll run the following command:
+
+```zsh
+slack create my-app -t slack-samples/bolt-python-starter-template -b future
```

Comment:
Some slightly odd formatting showing up here, not exactly sure why the extra lines are being produced..

![Screen Shot 2022-11-02 at 11 37 21 AM](https://user-images.githubusercontent.com/55667998/199573896-9e41184d-c9bf-4ae8-bff8-95d61ea44c52.png)

Reply:
Ive change the code type but Im not getting this behavior on my local 
<img width="1080" alt="Screen Shot 2022-11-02 at 4 08 54 PM" src="https://user-images.githubusercontent.com/25348381/199592696-65d903d2-d592-4ac1-a12d-4dc3ead8fe64.png">

Reply:
@srajiang @hello-ashleyintech is this behavior you also observe on the [live version of the website](https://slack.dev/bolt-python/concepts#message-sending)?